webdriver-spec.html

```diff
@@ -3039,16 +3040,18 @@
 <section>
 <h2>Element Retrieval</h2>
 
-<p>The <a>Find Element</a> and <a>Find Elements</a> <a>commands</a>
- allows look up of individual and a collection of <a>elements</a>, respectively.
+<p>The <a>Find Element</a> and <a>Find Elements</a> commands
+ allow look up of individual- and collections of <a>elements</a>.
+ Element retrieval searches are performed
+ using pre-order traversal of the document’s nodes
```

Comment:
This is just a description of how querySelectorAll works, but I’m happy to leave this undescribed if you think that’s a better idea?